pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -218,15 +219,20 @@ func (i *CommandInitOption) isExternalEtcdProvided() bool {
 }
 
 // Validate Check that there are enough flags to run the command.
-//
-//nolint:gocyclo
 func (i *CommandInitOption) Validate(parentCommand string) error {
 	if i.KarmadaAPIServerAdvertiseAddress != "" {
 		if netutils.ParseIPSloppy(i.KarmadaAPIServerAdvertiseAddress) == nil {
 			return fmt.Errorf("karmada apiserver advertise address is not valid")
 		}
 	}
 
+	switch i.ImagePullPolicy {
+	case string(corev1.PullAlways), string(corev1.PullIfNotPresent), string(corev1.PullNever), "":
```

Comment:
Never get a chance to be empty, right? Since it has default values,
Users are unlikely to set this flag to `image-pull-policy=""`.

Reply:
Yes, it is.